src/index.js

```diff
@@ -1,12 +1,23 @@
 const Hapi = require('hapi');
 const functionHelper = require('serverless-offline/src/functionHelper');
 
+const defaultConfig = {
+  port: 4000,
+  host: 'localhost',
+};
+
 class LambdaOffline {
   constructor(serverless, options) {
     this.service = serverless.service;
     this.options = options;
     this.serverless = serverless;
     this.serverlessLog = serverless.cli.log.bind(serverless.cli);
+    this.config = Object.assign(
+      {},
+      defaultConfig,
+      (this.service.custom || {})['serverless-offline'],
```

Comment:
We'll have port conflicts if we use `serverless-offline` + `serverless-offline-lambda` plugins at the same time because the server that emulates API gateway is a different one than for emulating Lambda invocations.

```yaml
custom:
  serverless-offline:
    port: 4000
```
Using the configuration above the lambda server will try to use the same port than API Gateway.

Reply:
I thought the same thing, but [this is the way the offline S3 plugin works as well](https://github.com/ar90n/serverless-s3-local/blob/master/index.js#L552).  I think it's actually a feature more than a bug, in that if you change the Serverless port only, things will fail because you then need to specify all the ports.  I can't think of any other reason why they'd have done it that way, unless there are other Serverless config options that plugins may need to inherit?

Reply:
Gotcha!! I didn't notice that on the S3 plugin before. Regarding the config options, nothing comes to mind so far, but I'll try to figure out what is missng and come back with something in this thread. thanks!!